ModuleSystem/ID/ID_scene_props_mb.py

```diff
@@ -1554,32 +1554,23 @@
 spr_sausages = 1553
 spr_boat_sail_on = 1554
 spr_boat_sail_off = 1555
-spr_tree_mirkwood_a = 1556
-spr_tree_mirkwood_b = 1557
-spr_tree_mirkwood_c = 1558
-spr_tree_mirkwood_roots_1 = 1559
-spr_tree_mirkwood_roots_2 = 1560
-spr_tree_mirkwood_roots_3 = 1561
-spr_tree_mirkwood_roots_4 = 1562
-spr_dolmen = 1563
-spr_orc_gate_destructible = 1564
-spr_save_compartibility1 = 1565
-spr_save_compartibility2 = 1566
-spr_save_compartibility3 = 1567
-spr_save_compartibility4 = 1568
-spr_save_compartibility5 = 1569
-spr_save_compartibility6 = 1570
-spr_save_compartibility7 = 1571
-spr_save_compartibility8 = 1572
-spr_save_compartibility9 = 1573
-spr_save_compartibility10 = 1574
-spr_save_compartibility11 = 1575
-spr_save_compartibility12 = 1576
-spr_save_compartibility13 = 1577
-spr_save_compartibility14 = 1578
-spr_save_compartibility15 = 1579
-spr_save_compartibility16 = 1580
-spr_save_compartibility17 = 1581
-spr_save_compartibility18 = 1582
-spr_save_compartibility19 = 1583
-spr_save_compartibility20 = 1584
+spr_save_compartibility1 = 1556
+spr_save_compartibility2 = 1557
+spr_save_compartibility3 = 1558
+spr_save_compartibility4 = 1559
+spr_save_compartibility5 = 1560
+spr_save_compartibility6 = 1561
+spr_save_compartibility7 = 1562
+spr_save_compartibility8 = 1563
+spr_save_compartibility9 = 1564
+spr_save_compartibility10 = 1565
+spr_save_compartibility11 = 1566
+spr_save_compartibility12 = 1567
+spr_save_compartibility13 = 1568
+spr_save_compartibility14 = 1569
+spr_save_compartibility15 = 1570
+spr_save_compartibility16 = 1571
+spr_save_compartibility17 = 1572
+spr_save_compartibility18 = 1573
+spr_save_compartibility19 = 1574
+spr_save_compartibility20 = 1575

```

Comment:
@In-Vain Looking at it again I can see that those weren't carriage returns, but a lot of space filling.